ci/release-qualification/pipeline.template.yml

```diff
@@ -58,8 +58,7 @@ steps:
       # 48h
       timeout_in_minutes: 2880
       agents:
-        # TODO: Move back to Hetzner when agent lost issues are fixed
-        queue: linux-aarch64-medium
+        queue: hetzner-aarch64-16cpu-32gb
```

Comment:
Minor (and slightly unrelated to this PR): I am not sure if it is a good idea to include the exact spec in the name (assuming we can influence this) or whether we should consider this an implementation detail. Changes to the spec will require us to rename an agent and references to it (which will not be the case with name suffixes like `medium`)... but maybe we want that.
Just a thought, no strong opinion.

Reply:
I did that intentionally because I always had to look up how many CPUs and GB our AWS-based agents have.